build.cake

```diff
@@ -249,21 +269,31 @@ Task("BuildEnvironment")
     CreateDirectory(dotnetFolder);
     var scriptPath = new FilePath($"{dotnetFolder}/{installScript}");
     DownloadFile($"{buildPlan.DotNetInstallScriptURL}/{installScript}", scriptPath);
-    if (!IsRunningOnWindows())
+    string installArgs = "";
+
+    if (IsRunningOnWindows())
+    {
+        installArgs = $"{buildPlan.DotNetChannel}";
+        if (!String.IsNullOrEmpty(buildPlan.DotNetVersion))
+            installArgs = $"{installArgs} -version {buildPlan.DotNetVersion}";
+        if (!buildPlan.UseSystemDotNetPath)
```

Comment:
Oh I just assumed that the guys followed standard *NIX argument naming convention where full argument names use "--". But it looks like they also recently added the Windows convention with "-version" and such.